R/utils_plotting.R

```diff
@@ -272,11 +278,12 @@ NULL
         geom_args$fill <- "grey70"
     }
     if (is.null(shape_by)) {
-        geom_args$shape <- 21
+        geom_args$shape <- shape
     }
     if (is.null(size_by)) {
         geom_args$size <- size
     }
+    geom_args$position <- position
```

Comment:
I think you can remove this from the internal helper again and just add the option in the required case. The logic of just adding a value can be done in the main internal function

Reply:
Removed